ceph_iscsi_config/lun.py

```diff
@@ -197,7 +197,7 @@ def required_features(cls):
 
 class LUN(object):
 
-    def __init__(self, logger, pool, image, size, allocating_host):
+    def __init__(self, logger, pool, image, size, allocating_host, ring_buffer_size=None):
```

Comment:
Well, you do have a point here -- it would be unclean (name wise) if we chose to add `command buffer size` later. I am in favor of accepting `ring_buffer_size` (as `cmd+data`) and figuring out the individual sizes -- is there a tried and tested method for that?